### DISCUSSION TITLE
Should we accept arrays of event handlers?

### DISCUSSION THREAD

**Discussion by Archmonger** [idom-team/idom#855] [Ideas]
Right now, events can only be single functions. However, in javascript it's possible to bind multiple event listeners to one event.

This discussion is to talk about whether this can/should be implemented in IDOM core during the v2 release cycle.

**Comment by rmorshea** [comment]
I think the best thing would be to provide a utility for merging event handlers. One [already exists](https://github.com/idom-team/idom/blob/main/src/idom/core/events.py#L206), but perhaps we could improve the interface. Previously I think IDOM had an `Event` object with an `add` and `extend` method for this purpose.

*For context, React does not have a way to pass arrays of event handlers.*
